### DISCUSSION TITLE
Lock Icon in Options dialog

### DISCUSSION THREAD

**Discussion by WildByDesign** [LesFerch/WinSetView#93] [Ideas]
@LesFerch I would suggest that you change the lock icon in the Options dialog to one that more closely mimics the shield icon for UAC. Your documentations covers it, which is great. But my first thought was that it represented a paid feature or a feature that was not available on my system. That is when I decided to dig into your documentation.

Anyway, just a suggestion. Thanks.

**Comment by LesFerch** [comment]
I may have made the wrong call on which Unicode character to use. The other candidates were: 👮, 🛡, ⛨, 🔑

I may just have to do the extra work to put the proper icon there as an inline image or convert the UAC shield to a vector graphic and then to a font glyph.

**Reply by WildByDesign** [reply]
That looks good. I haven’t checked for a while, but I am pretty sure that the open source Fluent UI System icon set by Microsoft has options for different shield types and elevation-related as well.

They provide the vector icons but also a set a Fonts compiled directly from their repo.

I am not sure if that would make it quicker for you to implement or not.

**Reply by LesFerch** [reply]
Here's the basic shield icon from the Segoe Fluent and MDL2 Assets fonts:
![image](https://github.com/LesFerch/WinSetView/assets/79026235/13345b7c-564b-4050-842e-3f4d1e4bf533)

It may be the best option for look and ease of implementation.

**Reply by WildByDesign** [reply]
That would work well. Quick and easy is good. It is definitely much easier to distinguish at first glance to understand what it represents.

**Comment by LesFerch** [comment]
It turns out that it's easier to use an inline image in this case, as it has to display correctly on Windows 7 as well. I'd need to create a subset font as I did for the other symbols. I ended up using this image:
![UAC11](https://github.com/LesFerch/WinSetView/assets/79026235/225ee063-08a6-4d9a-9b40-fc730f93ce6c)

**Reply by WildByDesign** [reply]
I just checked out your updated release build and it looks great. Thank you for the quick fix.

**Reply by LesFerch** [reply]
You're welcome and thanks the great suggestion.
